packages/sdk/src/sdk/config/staging.ts

```diff
@@ -5,101 +5,99 @@
       },
       {
-        endpoint: 'https://creatornode8.staging.audius.co',
-        delegateOwnerWallet: '0x8fcFA10Bd3808570987dbb5B1EF4AB74400FbfDA'
+        "endpoint": "https://creatornode8.staging.audius.co",
+        "delegateOwnerWallet": "0x8fcFA10Bd3808570987dbb5B1EF4AB74400FbfDA"
       },
       {
-        endpoint: 'https://creatornode12.staging.audius.co',
-        delegateOwnerWallet: '0x6b52969934076318863243fb92E9C4b3A08267b5'
+        "endpoint": "https://creatornode12.staging.audius.co",
+        "delegateOwnerWallet": "0x6b52969934076318863243fb92E9C4b3A08267b5"
       },
       {
-        endpoint: 'https://creatornode5.staging.audius.co',
-        delegateOwnerWallet: '0xDC2BDF1F23381CA2eC9e9c70D4FD96CD8645D090'
+        "endpoint": "https://creatornode5.staging.audius.co",
+        "delegateOwnerWallet": "0xDC2BDF1F23381CA2eC9e9c70D4FD96CD8645D090"
       },
       {
-        endpoint: 'https://creatornode6.staging.audius.co',
-        delegateOwnerWallet: '0x68039d001D87E7A5E6B06fe0825EA7871C1Cd6C2'
+        "endpoint": "https://creatornode6.staging.audius.co",
+        "delegateOwnerWallet": "0x68039d001D87E7A5E6B06fe0825EA7871C1Cd6C2"
       },
       {
-        endpoint: 'https://creatornode7.staging.audius.co',
-        delegateOwnerWallet: '0x1F8e7aF58086992Ef4c4fc0371446974BBbC0D9F'
+        "endpoint": "https://creatornode7.staging.audius.co",
+        "delegateOwnerWallet": "0x1F8e7aF58086992Ef4c4fc0371446974BBbC0D9F"
       },
       {
-        endpoint: 'https://creatornode9.staging.audius.co',
-        delegateOwnerWallet: '0x140eD283b33be2145ed7d9d15f1fE7bF1E0B2Ac3'
+        "endpoint": "https://creatornode9.staging.audius.co",
+        "delegateOwnerWallet": "0x140eD283b33be2145ed7d9d15f1fE7bF1E0B2Ac3"
       },
       {
-        endpoint: 'https://creatornode11.staging.audius.co',
-        delegateOwnerWallet: '0x4c88d2c0f4c4586b41621aD6e98882ae904B98f6'
+        "endpoint": "https://creatornode11.staging.audius.co",
+        "delegateOwnerWallet": "0x4c88d2c0f4c4586b41621aD6e98882ae904B98f6"
       }
     ],
-    antiAbuseOracleNodes: {
-      endpoints: ['https://antiabuseoracle.staging.audius.co'],
-      registeredAddresses: [
-        '0x00b6462e955dA5841b6D9e1E2529B830F00f31Bf',
-        '0x57B57efFA54ba37DBF8A06B9c42E7611e84BDe6F',
-        '0xF617bbc0913bAE0a13f6D4A19eCDE5Aa07B0fF0A'
+    "antiAbuseOracleNodes": {
+      "endpoints": [
+        "https://antiabuseoracle.staging.audius.co"
+      ],
+      "registeredAddresses": [
+        "0x00b6462e955dA5841b6D9e1E2529B830F00f31Bf",
+        "0x57B57efFA54ba37DBF8A06B9c42E7611e84BDe6F",
+        "0xF617bbc0913bAE0a13f6D4A19eCDE5Aa07B0fF0A"
       ]
     },
-    identityService: 'https://identityservice.staging.audius.co'
+    "identityService": "https://identityservice.staging.audius.co"
   },
-  acdc: {
-    entityManagerContractAddress: '0x1Cd8a543596D499B9b6E7a6eC15ECd2B7857Fd64',
-    chainId: 1056801
+  "acdc": {
```

Comment:
at first i was confused by the quotes. someone must have ctrl+s'd this and gotten autoformatted before haha

Reply:
i think so!